docs/src/Tooltip.doc.js

```diff
@@ -39,6 +39,12 @@ card(
         description:
           'String that is shown as addition information in a tooltip bubble to describe the child. Always localize the text.',
       },
+      {
+        name: 'link',
+        type: 'React.Node',
+        required: false,
+        description: 'A link element',
```

Comment:
@anniejiwon could we be more descriptive? E.g. `A link which will show on the bottom of a tooltip`